scripts/check_frontend_test_coverage.py

```diff
@@ -46,7 +46,11 @@
     'Base.ts',
     'ck-editor-4-rte.component.ts',
     'ck-editor-4-widgets.initializer.ts',
-    'collection-editor-navbar-breadcrumb.directive.ts',
+    'collection-details-editor.component.ts',
+    'collection-editor-navbar-breadcrumb.component.ts',
```

Comment:
Hmmm, can we not add to this file? Why do we need to add new stuff here?

Reply:
done